pyquil/paulis.py

```diff
@@ -33,6 +34,10 @@
 from collections.abc import Sequence
 import warnings
 
+ExpressionValueDesignator = Union[int, float, complex]
```

Comment:
These type aliases are importable, so you should be able to import this from `quilatom.ExpressionValueDesignator` rather than redefining here.

However, something doesn't quite seem right about reusing the `ExpressionValue` name here. That name was coined to describe the types of values that might be involved in a `pyquil.Expression`. I'm not too familiar with how this `paulis` module fits in, so maybe the types here are necessarily related to the `pyquil.Expression` value types, but it isn't obvious to me that should be the case.

Reply:
I have imported from `pyquil.quilatom`, but leaving the comment unresolved for now.